src/core_modules/capture-core/components/WidgetEventEdit/EditEventDataEntry/editEventDataEntry.epics.js

```diff
@@ -189,20 +191,19 @@ export const saveEditedEventFailedEpic = (action$: InputObservable, store: Redux
             const meta = action.meta;
             const viewEventPage = state.viewEventPage;
             const eventContainer = viewEventPage.loadedValues.eventContainer;
-            const orgUnit = state.organisationUnits[eventContainer.event.orgUnitId];
             if (eventContainer.event && eventContainer.event.attributeCategoryOptions) {
                 eventContainer.event.attributeCategoryOptions =
                     convertCategoryOptionsToServer(eventContainer.event.attributeCategoryOptions);
             }
-            let actions = [updateEventContainer(eventContainer, orgUnit)];
+            let actions = [];
 
             if (meta.triggerAction === enrollmentSiteActionTypes.ROLLBACK_ENROLLMENT_EVENT) {
                 actions = [...actions, rollbackEnrollmentEvent(eventContainer.event.eventId)];
             }
             if (meta.triggerAction === enrollmentEditEventActionTypes.EVENT_SAVE_ENROLLMENT_COMPLETE_ERROR) {
```

Comment:
I removed the unnecessary spread syntax on `actions`. Thanks for the feedback!

Reply:
It is more efficient to use `else if` as well here (because then the second condition will not get evaluated whenever the first one evaluates to `true`)

Reply:
Good point! The code should be updated now. Thanks!